_config.yml

```
@@ -973,3 +973,10 @@ podcast:
       file_byte_length: "33427019"
       duration: "00:34:53"
       credits: "Jon Allured, Ash Furrow, & Steve Hicks"
+    - title: "5. The Structure of Artsy's Product Team"
```

nonblocking

```suggestion
    - title: "5: The Structure of Artsy's Product Team"
```

This is ridiculously nit-picky and you can totally ignore it, just aligning with previous episodes.

---

Oops good call 👍